File: 2160_Minimum_Sum_of_Four_Digit_Number_After_Splitting_Digits/solution.c
int compare(const void* a, const void* b) { return (*(int*)a - *(int*)b); }
int minimumSum(int num) {
    int nums[4];
    for (int i = 0; i < 4; i++) nums[i] = num % 10, num /= 10;
    qsort(nums, 4, sizeof(int), compare);
    return (nums[0] * 10 + nums[3]) + (nums[1] * 10 + nums[2]);
}

File: 2160_Minimum_Sum_of_Four_Digit_Number_After_Splitting_Digits/solution.py
# class Solution:
#     def minimumSum(self, num: int) -> int:
#         nums = []
#         while num != 0:
#             nums.append(num % 10)
#             num //= 10

#         nums.sort()

#         num_1 = nums[0] * 10 + nums[3]
#         num_2 = nums[1] * 10 + nums[2]

#         return num_1 + num_2

class Solution:
    def minimumSum(self, num: int) -> int:
        nums = sorted(str(num))
        return int(nums[0] + nums[3]) + int(nums[1] + nums[2])